bazel/grpc_deps.bzl

```diff
@@ -66,11 +66,6 @@ def grpc_deps():
         actual = "@com_github_google_googletest//:gtest",
     )
 
-    native.bind(
```

Comment:
is gmock going away with the new version of gtest? It would be good to understand why it's being removed here.

Reply:
In v1.8.1, gtest includes gmock. https://github.com/google/googletest/blob/release-1.8.1/BUILD.bazel#L56.